portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,22 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 /**
- * Adds a random greeting to the page.
+ * Leads user to a selected project.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function chooseFeaturedProj() {
+  const urls =
+      ['https://github.com/georgezheng999/Depmap-Data-Scraping', 'https://github.mit.edu/MEDSL/primary-precincts', 'https://github.com/georgezheng999/Pac-Man', '', 'https://github.com/georgezheng999/Chess-Two-Player', 'https://github.com/georgezheng999/APCS-Spring-Projects'];
```

Comment:
> '',

This element in the array is empty. Were you planning to add the link later? If so, it would be good to leave a TODO above.

e.g.
```
// TODO: Add the missing link.
const urls =
```

Reply:
This is a bit hacky, but this url corresponds to the portfolio itself -- in the browsers I've tested it on, using an empty string will simply refresh the page as desired. I wasn't able to find a simpler way to achieve this without having to treat this as a special case; I'll add a comment clarifying this.